.github/workflows/release.yml

```diff
@@ -0,0 +1,37 @@
+name: Release charts
+on:
+  push:
+    branches:
+      - master
+      
+env:
+    HELM_VERSION: v3.13.1
+
+jobs:
+  release:
+    name: Release charts
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write # to push chart release and create a release (helm/chart-releaser-action)
+      id-token: write # needed for keyless signing
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0 # to pull repo with all history
+
+      - name: Configure Git
+        run: |
+          git config user.name "$GITHUB_ACTOR"
+          git config user.email "$GITHUB_ACTOR@users.noreply.github.com"
```

Comment:
Please follow the Google's convention for bash scripts to make the environment variables more visible:

```
"${GITHUB_ACTOR}"
"${GITHUB_ACTOR}@users.noreply.github.com"
```

In addition,  adding `set -ue` in the first line of the script will help troubleshooting issues, if GitHub Actions' API changes in the future.

Reply:
done